tests/foreman/cli/test_variables.py

```diff
@@ -283,7 +288,8 @@ def test_positive_update_variable_puppet_class(self):
         smart_variable = make_smart_variable({
             'puppet-class': self.puppet['name']})
         self.assertEqual(smart_variable['puppet-class'], self.puppet['name'])
-        new_puppet = Puppet.info({u'name': 'ntp::config'})
+        new_puppet = Puppet.info(
+            {u'name': self.puppet_modules[1]['name']})
```

Comment:
This may be overkill but maybe you could write a function that would randomly pick a module for you as long as it is not the one you want to 'keep'?

```python
def module_roulette(modules, module_to_keep=None):
    # If module_to_keep then randomly pick a value from modules excluding it
```